firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+ZoomPage.swift

```diff
@@ -33,7 +33,9 @@ extension BrowserViewController {
             .isActive = true
         zoomPageBar.applyTheme(theme: themeManager.currentTheme)
 
-        updateViewConstraints()
+        if UIDevice.current.userInterfaceIdiom != .pad {
+            updateViewConstraints()
+        }
```

Comment:
What happens if we're in multitasking mode on iPad? 🤔

Reply:
@lmarceau 

Everything is working fine, after I tested it.

Reply:
@dicarobinho I'm slightly confused by this change but maybe I'm just misunderstanding; at a glance it seems like we're just outright skipping the constraint configuration for iPad idiom, but isn't the real issue that the constraints themselves are incorrect? IOW, shouldn't we be fixing the constraints in `updateViewConstraints` rather than just skipping that call entirely? Or are we really not supposed to be calling `updateViewConstraints` at all on iPad?

Reply:
@mattreaganmozilla 

Hey, matt. updateViewConstraints() method is called from many places in code. Updating it only for the ZoomBar, we should make sure that everything else is not impacted negatively. So, while everything with ZoomBar on iPad is working as expected after removing updateViewConstraints() call, I am still going forward with this approach.